source/index.js

```diff
@@ -12,9 +12,11 @@ var renderer = Pixi.autoDetectRenderer(Reference.GAME_WIDTH, Reference.GAME_HEIG
 renderer.backgroundColor = 0x222222
 renderer.roundPixels = true
 
+renderer.view.id = "gameCanvas"
 document.getElementById("mount").appendChild(renderer.view)
 window.game = new GameContainer()
 game.addChild(new Junkership())
+game.addChild(new Junkership())
```

Comment:
Ooooh, two `Junkerships`! We need to give their constructor some parameters, so we can pass them in separate data, and therefore make them separate distinct entities (instead of both of them being stacked on top of each other)